frontend/src/app.vue

```diff
@@ -141,15 +164,16 @@ const app = {
       this.userUpdated = false;
       await this.$store.dispatch('incrementLoadingOverlayCountForceReload', 1);
       try {
+        const summary = await window.api.loadSummary();
```

Comment:
Could it be useful to have an interface for `summary`?

Reply:
`loadSummary` is already typed in `api.ts`, so there should already be type inference for the `summary` object!